repository/model_repository.go

```diff
@@ -1,16 +1,34 @@
 package repository
 
 import (
-	"dummy/mock"
 	"dummy/model"
 )
 
-func GetModels() []model.Model {
-	models := mock.MockModels()
-	return models
+func GetModels() ([]model.Model, error) {
+	queryResult, err := Repository.DB.Query("SELECT id, attribute1 FROM model")
+	if err != nil {
+		return nil, err
+	}
+
+	defer queryResult.Close()
+
+	var models []model.Model
+	for queryResult.Next() {
+		var m model.Model
+		queryResult.Scan(&m.Id, &m.Attribute1)
+		models = append(models, m)
+	}
+
+	return models, nil
 }
 
-func GetModel(id string) model.Model {
-	m := mock.MockModel(id, "ASDASD")
-	return m
+func GetModel(id string) (model.Model, error) {
+	queryResult := Repository.DB.QueryRow("SELECT id, attribute1 FROM model WHERE id = $1", id)
+	var m model.Model
+	err := queryResult.Scan(&m.Id, &m.Attribute1)
+	return m, err
+}
+
+func DeleteModel(id string) {
+
 }
```

Comment:
Empty method, better delete / throw panic.
```suggestion
func DeleteModel(id string) {
	panic("unimplemented")
}
```